dev/tests/integration/testsuite/Magento/CatalogInventory/Model/StockItemSave/OnProductCreate/ByProductRepository/ByStockItemTest.php

```diff
@@ -61,7 +61,11 @@ class ByStockItemTest extends \PHPUnit\Framework\TestCase
         ProductInterface::NAME => 'Simple',
         ProductInterface::SKU => 'simple',
         ProductInterface::PRICE => 100,
-        ProductInterface::EXTENSION_ATTRIBUTES_KEY => [],
+        ProductInterface::EXTENSION_ATTRIBUTES_KEY => [
```

Comment:
We can't assign `ProductInterface::EXTENSION_ATTRIBUTES_KEY` for all of testcases

It will be applicable if we set stock data via product model directly
We have testcase `\Model\StockItemSave\OnProductCreate\ByProductRepository\ByStockItemTest::testSave` for checking behaviour

But in the other case like in `testSaveManuallyCreatedStockItem` or `testAutomaticallyStockItemCreating`
We MUST HAVE empty  `ProductInterface::EXTENSION_ATTRIBUTES_KEY`